app/models/ingredient_item.rb

```diff
@@ -0,0 +1,2 @@
+class IngredientItem < ApplicationRecord
```

Comment:
We may want to rename this to InventoryItem for clarity. We would also then want to update the names of the controller, tests, etc.